### ISSUE TITLE
Can I use tauri in an existing native project?

### ISSUE THREAD

**Issue by tanzhh-bingosoft** [wusyong/cargo-mobile#242]
Can I use tauri in an existing native project?

**Comment by ahqsoftwares** [comment]
probably not

**Comment by FabianLars** [comment]
yeah, if you some kind of magician i'm sure you can make it work (likely requires changes on tauri's side) but this is out of scope for us for the foreseeable future.

**Comment by ahqsoftwares** [comment]
In that case using ndk rs directly would be much better than forking this repo and editing it
